packages/collector/test/tracing/cloud/aws-sdk/v2/dynamodb/test.js

```diff
@@ -151,6 +151,7 @@ mochaSuiteFn('tracing/cloud/aws-sdk/v2/dynamodb', function () {
       });
 
       if (!withError) {
+        expect(spans.length).to.equal(3);
```

Comment:
idea: Please push the asserts straight to main and rebase against main. Adding them as part of the PR won't reveal bugs.